### ISSUE TITLE
[Beta Feedback]: Nifty.ink

### ISSUE THREAD

**Issue by igorbarinov** [omni/omnibridge-nft-ui#54]
Satisfied: 
Somewhat satisfied

Bugs/Issues:
When starting on xDai Chain and transferring to Rinkeby.  The UI provides note "The claim process requires 2 transactions, one on xDai Chain and one on Rinkeby Testnet."

When really, the use of the bridge requires two txns, not the Claim Process.  The Claim process will be one txn.  

And then when i sent the NFT across the bridge, I was expecting to Claim, but instead received the note: "The tokens were already claimed.  Check for your token in Rinkeby Testnet."

So overall, this is a bit unclear and incorrect.   

Additional comments:

Attachment:

**Comment by dan13ram** [comment]
#53 will make this clearer. The claim process will be disabled for Rinkeby-xDai.

But for Eth-Mainnet we will still be showing that message. But maybe we can tweak it a bit:
```
The bridging process requires 2 transactions, one on xDai Chain and one on Ethereum Mainnet.
```
Does that sound alright @igorbarinov  ?

**Comment by akolotov** [comment]
> #53 will make this clearer. The claim process will be disabled for Rinkeby-xDai.
> 
> But for Eth-Mainnet we will still be showing that message. But maybe we can tweak it a bit:
> 
> ```
> The bridging process requires 2 transactions, one on xDai Chain and one on Rinkeby Testnet.
> ```
> 
> Does that sound alright @igorbarinov ?

I beleive we need to make a proper fix for #53. Don't think that a suggested workaround makes sense since the functionality will be changed later anyway.

**Comment by dan13ram** [comment]
Yeah I said that we shall do the proper fix in #53, while tweaking the messaging a bit for xDai => Mainnet.

**Comment by akolotov** [comment]
The claim issue for the Rinkeby-xDai direction was addressed as part of the work under #53
